.github/workflows/security.yaml

```diff
@@ -72,10 +72,19 @@ jobs:
         with:
           distribution: "goreleaser-pro"
           version: "2.3.2"
-          args: "release --clean --split --snapshot --single-target --skip=chocolatey"
```

Comment:
There's no chocolatey so there's no reason for it

Reply:
Where did it go?

Reply:
It's in `.goreleaser.windows.yaml`, which isn't used by this command.